storagetransfer/aws_s3_compatible_source_request_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import backoff
+from google.api_core.exceptions import RetryError
+from google.api_core.exceptions import ServiceUnavailable
+from google.cloud.storage import Bucket
+from google.cloud.storage.bucket import _API_ACCESS_ENDPOINT
+from google.cloud.storage_transfer_v1.types import transfer_types
+
+import aws_s3_compatible_source_request
+
+
+@backoff.on_exception(backoff.expo, (RetryError, ServiceUnavailable,),
+                      max_time=60)
+def test_aws_s3_compatible_source_request(
+        capsys, project_id: str, job_description_unique: str,
+        agent_pool_name: str, source_bucket: Bucket,
+        posix_root_directory: str, destination_bucket: Bucket):
```

Comment:
Extra indentation isn't necessary.

```suggestion
def test_aws_s3_compatible_source_request(
    capsys,
    project_id: str,
    job_description_unique: str,
    agent_pool_name: str,
    source_bucket: Bucket,
    posix_root_directory: str,
    destination_bucket: Bucket,
) -> None:
```

Reply:
Updated